### ISSUE TITLE
Unable to board Helicopter in Estoni

### ISSUE THREAD

**Issue by nemuc** [ja2-stracciatella/ja2-stracciatella#800]
The Helicopter and Red are both in Estoni (The Helicopter base is still in Drassen). When trying to board the helicopter via strategic view the menu point Vehicle is greyed out.

| JA2S version | Game version        | Vanilla bug? | Used to work? |
| ------------ | ------------------- | ------------ | ------------- |
|  v0.16.1 | JA2 Gold English  |    ?    |    yes     |

- operating system is Debian 9 Stretch
- save attached
[SaveGame11.sav.zip](https://github.com/ja2-stracciatella/ja2-stracciatella/files/2931412/SaveGame11.sav.zip)

**Comment by nemuc** [comment]
This is even worse:
Skyrider doesn't accept movement orders anymore. And he doesn't return to Drassen when the time passes. Upon selection he states: `I am chopperless, cap. Can't do much for ya.`

I suspect, the issue might have something to do with Estoni.

Additional Info:
I spoke to Jake and to Skipper but not Maddog. I haven't been to Tixa yet.

PS:
This brings my "Dead is Dead" to an end, because the backups have the same issue and without helicopter it isn't fun. I really liked that Dead is Dead did all the saving for me and I could stop playing in the middle of a battle.

**Comment by lynxlynxlynx** [comment]
Part of this sounds like #723, but maybe there's more to it.

**Comment by nemuc** [comment]
Thanks for linking to that bug. Indeed, these bugs are similar. However, I have no sign that Estoni would be enemy controlled.

Could anybody point out to what exactly @misanthropos refers to with:
`EnemyControlled` (Sector)
`CanHelicopterTakeOff `

I followed [the debugging tips](https://github.com/ja2-stracciatella/ja2-stracciatella/wiki/Tips-and-Tricks-for-Developers) and using the Quest Debug Screen (F11) I only identified these as relevant facts:
31 Current Secor is safe `TRUE`
43 Skyrider is close to his chopper in Drassen `TRUE`
126 Helicopter is in working condition (was FALSE =Changed to `TRUE`)
130 Helicoper was destroyed is `FALSE`
159 Skyrider left in the wrong sector is `FALSE`
164 Shank is part of squad is `FALSE`
167 Shank (67) is not in sector is `TRUE`
227 vehicle is present is `FALSE` (might refer to the Hummer)
277 Estoni refuelling possible `FALSE` (changing to TRUE didn't solve the problem)

**Comment by nemuc** [comment]
I solved it by cheating an enemy into existence and executing him. After I won this "battle" I was able to use the helicopter again.

I suggest closing  this as a duplicate of #723: 

1. approached Estoni by single merc in helicopter
2. sector is somehow EnemyControlled without actual enemy presence
3. helicopter again available after winning a battle in Estoni and thus regaining the control of the sector
